index.css

```diff
@@ -8,6 +12,46 @@ main {
   background: url(IMG/header-background-mobile.png);
   border-bottom-left-radius: 100px;
   background-repeat: no-repeat;
+  position: relative;
+}
+
+.js-menu {
+  position: absolute;
+  background-color: #6070ff;
+  opacity: 0.6;
+  height: 700px;
+  width: 375px;
+  padding-top: 20px;
+  transition: opacity 500ms ease;
+}
+
+#canceli {
+  margin-left: 315px;
+}
```

Comment:
Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_5](https://user-images.githubusercontent.com/69638013/161509772-8d06045b-1c2e-429a-a19f-5a13841832a9.png) | ![bad1](https://user-images.githubusercontent.com/79058364/161735060-55b858be-54ae-42e1-8ed5-d65061074b53.png)

- [OPTIONAL ] What if you use `position: fix` instead of `absolute` ? That will help preventing your mobile navigation menu to scroll, kindly change this

- [ OPTIONAL] Kindly make the under background blurry when the mobile navigation is open, this will make the mobile nav text more visible to the users 👍🏿 

- [OPTIONAL ] It would be better for the background not to be scrollable when the mobile nav menu is open . you can use the CSS property `overflow: hidden;` to prevent that.

- [ ]  Adding these features will make your design look good and look exactly like Figma's design, I highly recommend you do this in the meantime, I'll approve this project since the previous reviewer is skipping this